homewizard_energy/v1/__init__.py

```diff
@@ -120,7 +133,7 @@ async def identify(
         """Send identify request."""
 
         if self._device is not None and self._device.supports_identify() is False:
-            raise UnsupportedError("State is not supported")
+            raise UnsupportedError("Identify is not supported")
```

Comment:
While being incorrect, this is irrelevant for this PR
```suggestion
            raise UnsupportedError("State is not supported")
```